demos/src/demo.scss

```diff
@@ -8,18 +8,5 @@ html {
 
 body {
 	margin: oGridGutter();
-}
-
-.demo-real-life {
-	padding: 0;
-
-	body {
-		margin: 0;
-	}
-
-	h2,
-	h3 {
-		@include oFormsGroup; // Indent
-		margin-bottom: 1em; // Set smaller margin
-	}
+	background-color: #fff1e0;
```

Comment:
I think this can be replaced with ft-pink from o-colors -- https://github.com/Financial-Times/o-colors/blob/11395ed083a2c194a728201e4215c3d43d7b8182/src/scss/_palette.scss#L35